integrations/groq/integration.definition.ts

```diff
@@ -1,6 +1,13 @@
 import { llm } from '@botpress/common'
 import { IntegrationDefinition, z } from '@botpress/sdk'
 
+const model = z
+  .enum(['llama3-8b-8192', 'llama3-70b-8192', 'mixtral-8x7b-32768', 'gemma-7b-it'])
+  .describe('Model identifier to be used for content generation.')
```

Comment:
nitpick, just to make it simpler to understand

```suggestion
  .describe('Model to use for content generation')
```